submodules/eks/variables.tf

```diff
@@ -195,6 +195,11 @@ variable "eks" {
       username_claim                = optional(string, null)
       username_prefix               = optional(string, null)
     })), [])
+    irsa = optional(object({
+      enabled                    = bool
+      namespace_service_accounts = list(string)
+      role_name                  = optional(string)
+    }))
```

Comment:
This structure assumes only a single role is necessary for Domino to function when we'll need to be more flexible over time, e.g.:
* backups will want a limited role that can only upload to S3
* EBS CSI driver will need a role that can manage volumes
* fluentd and nucleus actually have different requirements: fluentd only needs to read/write to S3 while nucleus needs S3 and EC2 API access.